src/Components/CardList/Card/index.jsx

```diff
@@ -1,71 +1,55 @@
-import React, { useState, Fragment } from 'react';
+import React, { useState } from 'react';
 import CardHeader from './CardHeader';
 import CardBody from './CardBody';
 import withLoadingDelay from '../../../HOC/withLoadingDelay';
 import PropTypes from 'prop-types';
 import './style.css';
 
 const Card = props => {
-  const [newState, setNewState] = useState({
-    ...props.card
-  });
+  const [newState, setNewState] = useState({...props.card});
 
   const selectItem = () => {
-    props.onSaveChanges({
-      ...newState,
-      chooseСardFlag: !props.card.chooseСardFlag
-    })
+    props.onSaveChanges({...newState, chooseСardFlag: !props.card.chooseСardFlag})
   };
 
   const addNewText = event => {
-    setNewState({
-      ...newState,
-      [event.target.name]: event.target.value
-    });
+    setNewState({...newState, [event.target.name]: event.target.value});
   };
 
   const editModeOn = () => {
-    props.onSaveChanges({
-      ...newState,
-      editModeFlag: true,
-      chooseСardFlag: false
-    });
-  }
+    props.onSaveChanges({...newState, editModeFlag: true, chooseСardFlag: false});
+    setNewState({...newState, editModeFlag: true, chooseСardFlag: false});
+  };
 
   const saveChenges = () => {
-    props.onSaveChanges({
-      ...newState,
-      editModeFlag: false
-    });
+    props.onSaveChanges({...newState, editModeFlag: false});
   };
 
   const abortChanges = () => {
-    props.onSaveChanges({
-      ...props.card,
-      editModeFlag: false
-    });
-    setNewState({
-      ...props.card,
-      editModeFlag: false
-    })
-  }
+    props.onSaveChanges({...props.card, editModeFlag: false});
+    setNewState({...props.card, editModeFlag: false})
+  };
+
+  const toFullCardPage = () => {
+    if (!newState.editModeFlag && !props.onView) {
```

Comment:
Why don't you allow to open Card page in View Mode? Page should be accessible, simply in a view mode only.  Think about moving View Mode state to redux as well.

Reply:
Fixed